Changelog.md

```diff
@@ -1,8 +1,9 @@
-## 4.3.1 - Unreleased
+## 4.4.0 - Unreleased
 
 ### Changed
 
--   Notarize with stricter entitlements for macOS
+-   Notarize with stricter entitlements for macOS.
+-   Minor fixes and improvements.
```

Comment:
Mentioning all the changes listed in Changelog.minor.md. We do not do like this, but maybe still worth informing users that there are "other changes", but without details in this case? @bihanssen

Reply:
agreed